contracts/neutron_interchain_queries/src/msg.rs

```diff
@@ -66,10 +73,17 @@ pub enum QueryMsg {
     StakingValidators { query_id: u64 },
     GovernmentProposals { query_id: u64 },
     GetDelegations { query_id: u64 },
+    Cw20Balance { query_id: u64 },
     GetRegisteredQuery { query_id: u64 },
     GetRecipientTxs { recipient: String },
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct Cw20BalanceResponse {
```

Comment:
if we create a new_register_cw20_balance_query_msg in the helpers package, we should move this one to the helpers package as well